src/modules/Dom.js

```diff
@@ -0,0 +1,5 @@
+const modal = document.querySelector('.popup-container');
+const comment = document.querySelector('.comment');
+const closeBtn = document.querySelector('.close-window');
```

Comment:
- [ ]  `const closeBtn = document.querySelector('.close-window');`  you never used this in your code and its looks like and old code , so kindly remove it since it does not affect the code